docs/settings/reporting-settings.asciidoc

```diff
@@ -104,15 +104,15 @@ security is enabled, `xpack.security.encryptionKey`.
 [cols="2*<"]
 |===
 | `xpack.reporting.queue.pollInterval`
-  | Specifies the number of milliseconds that the reporting poller waits between polling the
-  index for any pending Reporting jobs. Defaults to `3000` (3 seconds).
+  | (time value) Specifies the time that the reporting poller waits between polling the
+  index for any pending Reporting jobs. Defaults to `3s`.
 
 | [[xpack-reporting-q-timeout]] `xpack.reporting.queue.timeout`
-  | How long each worker has to produce a report. If your machine is slow or under
-  heavy load, you might need to increase this timeout. Specified in milliseconds.
+  | (time value) How long each worker has to produce a report. If your machine is slow or under
```

Comment:
@elastic/kibana-docs is there a good way to link `time value` to this doc: https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html#time-units